### PR TITLE
desktop: Detect OS architecture for Linux download

### PR DESCRIPTION
**Pull request by feross**
Change the download page to link to the 32-bit Linux download.

Use the soon-to-be-published `arch` package (thanks for the npm package
name @Raynos!) to detect the OS architecture and conditionally change
the link to the 64-bit version.

https://github.com/feross/arch

### CODE REVIEW

**Comment by feross** [comment]
I also went through and made sure all instances of `document.querySelectorAll()` are wrapped by an `Array.from()` and filed a Chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=647860

Sheesh. cc @dcposch

**Comment by feross** [comment]
Can I get a review? @dcposch @mathiasvr @Flet @ungoldman @demoneaux

File: client/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // View-specific code
 var pathname = window.location.pathname
 if (pathname === '/') require('./views/home')()
-if (pathname === '/create') require('./views/create')()
+if (/\/create\/?/.test(pathname)) require('./views/create')()
+if (/\/desktop\/?/.test(pathname)) require('./views/desktop')()
```
Comment (dcposch) [inline client/index.js]: How did this work before?

**Comment by dcposch** [comment]
Looks like the npm package is published
https://www.npmjs.com/package/arch

**Comment by dcposch** [comment]
What's this fail? It won't merge even tho it's reviewed.

### Step 1

screen shot 2016-09-17 at 9 20 22 pm

### Step 2

screen shot 2016-09-17 at 9 20 09 pm

**Comment by ungoldman** [comment]
You gotta push the button harder

File: client/index.js (hunk 1 above)
Comment (feross) [inline client/index.js]: It didn't really. The `/create` page was a work-in-progress from the very first version of the site (years ago!). It should probably be removed!

**Comment by feross** [comment]
@ungoldman 😆

**Pull request merged**